dvc/command/experiments.py

```diff
@@ -710,12 +710,16 @@ def add_parser(subparsers, parent_parser):
     experiments_run_parser.add_argument(
         "--continue",
         type=str,
+        nargs="?",
         default=None,
+        const=":last",
```

Comment:
Use `:` prefix to ensure we can't collide with a valid branch or tag name (currently branch names are only auto-generated but we may support user named branches or tags in the future)